simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/transactionctrlr/StopTxOnEVSideDisconnectVariableAccessor.java

```diff
@@ -0,0 +1,95 @@
+package com.evbox.everon.ocpp.simulator.station.component.transactionctrlr;
+
+import com.evbox.everon.ocpp.common.CiString;
+import com.evbox.everon.ocpp.simulator.station.Station;
+import com.evbox.everon.ocpp.simulator.station.StationStore;
+import com.evbox.everon.ocpp.simulator.station.component.ReportDataGenereator;
+import com.evbox.everon.ocpp.simulator.station.component.variable.SetVariableValidator;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableAccessor;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableGetter;
+import com.evbox.everon.ocpp.simulator.station.component.variable.VariableSetter;
+import com.evbox.everon.ocpp.simulator.station.component.variable.attribute.AttributePath;
+import com.evbox.everon.ocpp.simulator.station.component.variable.attribute.AttributeType;
+import com.evbox.everon.ocpp.v201.message.centralserver.Attribute;
+import com.evbox.everon.ocpp.v201.message.centralserver.GetVariableResult;
+import com.evbox.everon.ocpp.v201.message.centralserver.GetVariableStatus;
+import com.evbox.everon.ocpp.v201.message.station.ReportData;
+import com.evbox.everon.ocpp.v201.message.station.VariableAttribute;
+import com.evbox.everon.ocpp.v201.message.station.VariableCharacteristics;
+import com.google.common.collect.ImmutableMap;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static com.evbox.everon.ocpp.v201.message.station.Data.BOOLEAN;
+import static com.evbox.everon.ocpp.v201.message.station.Data.INTEGER;
+import static com.evbox.everon.ocpp.v201.message.station.Mutability.READ_ONLY;
+import static com.evbox.everon.ocpp.v201.message.station.Mutability.READ_WRITE;
+
+public class StopTxOnEVSideDisconnectVariableAccessor extends VariableAccessor {
```

Comment:
VariableAccessor implementations have a lot of boilerlate without much deviations, maybe we can move it to the base class? 

filed names and maybe actualvalue and reportdata methods are different... rest looks very same

Reply:
For the report methods, because each variable have a different type/name/value.. I tried to build these variables individually but call the common part for generating the report from `ReportDataGenereator.generateReportData`
for other existing methods they are mostly abstract and implemented differently depending on the variable and whether we fully support it or not